web/packages/teleport/src/Discover/Shared/SetupAccess/useUserTraits.ts

```diff
@@ -222,14 +242,17 @@ export function useUserTraits(props: AgentStepProps) {
 
       default:
         throw new Error(
-          `useUserTraits.ts:updateResourceMetaDynamicTraits: resource kind ${props.resourceSpec.kind} is not handled`
+          `useUserTraits.ts:updateResourceMetaDynamicTraits: resource kind ${resourceSpec.kind} is not handled`
```

Comment:
i can't use `assertUnreachable` because I didn't define all the cases for the enum i'm using (it'll complain until i define all the enum vals as cases). this function is more like "add more cases as we support more resources" and atm we only support setting traits for small select num resources.